source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -40,10 +74,24 @@ Other Operator Changes
 Deprecations & Removals
 ***********************
 
+- The existing "Content Libraries" feature has been renamed to "Legacy Libraries" and will be deprecated in the next release (Teak), and removed entirely in release after Teak in favor of the new "Content Libraries" feature. The Teak release will include a tool for migrating content from Legacy Libraries into new Content Libraries.
+- In `frontend-app-learner-dashboard <https://github.com/openedx/frontend-app-learner-dashboard>`_ 
+   - support for Optimizely has been removed along with the ProductRecommendations widget.
+   - `DEPR: Optimizely Support <https://github.com/openedx/frontend-app-learner-dashboard/issues/387>`_
+   - Removed the RecommendationsPanel widget
+   - `DEPR: RecommendationsPanel <https://github.com/openedx/frontend-app-learner-dashboard/issues/410>`_
+- In edx-platform:
+   - ``commerce-coordinator`` related code has been removed
+   - `PR: <https://github.com/openedx/edx-platform/pull/35527>`_
+- [UPCOMING] In Teak pre-design-tokens brand packages will no longer be supported. With design tokens, theme authors will instead override core Paragon tokens by defining their own JSON tokens that get deep merged alongside the core Paragon tokens, thus overriding any tokens that were defined by the theme author. See `the associated DEPR ticket for details <https://github.com/openedx/brand-openedx/issues/23>`_.
+- The cs_comments_service application is being replaced by forum. `[DEPR]: Replace cs_comments_service #437 <https://github.com/openedx/cs_comments_service/issues/437>`_
+- The Zooming Image Tool will be deprecated in Sumac. `[DEPR] Zooming Image Tool (HTML block template) #31436 <https://github.com/openedx/edx-platform/issues/31436>`_
 
 Developer Experience
 ********************
 
+- With the launch of the new Content Libraries feature, many new or updated APIs are now ready for developers to start using (though some are in beta and may change). This includes the entire `Learning Core API <https://github.com/openedx/openedx-learning/blob/main/openedx_learning/api/authoring.py>`_ (Components, Contents, Publishing, Collections), `Content Tagging API <https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/content_tagging/api.py>`_, `Content Libraries API <https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/content_libraries/api.py>`_, and `Learning Core XBlock API <https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/xblock/api.py>`_. Most of these APIs (other than tagging) are only useful in content libraries at the moment, but when courseware is eventually moved to Learning Core as well, the Learning Core APIs will be recommended way to work with all learning content in the platform, and the modulestore APIs will be deprecated.
+
 Researcher & Data Experiences
```

Comment:
@bmtcril anything to add about Aspects in the developer release notes?

Reply:
@crathbun428 was collecting notes, but I'm not sure if they belong here or where they were going.

Reply:
Right, Chelsea has user-facing notes. Are there things developers or operators (more technical audience) should know, as they upgrade from Redwood?

Reply:
Yep - [these feature release notes](https://docs.openedx.org/en/latest/community/release_notes/sumac/aspects_comp_dashboard.html) are live. I was planning on linking to this section of the operator release notes from the feature release notes. I had checked in with Brian about any changes to this section and it seems this section we had last time may still work as I believe all of the linked documents have been updated.
---
[Aspects](https://docs.openedx.org/projects/openedx-aspects/en/latest/index.html) is an analytics system for the Open edX platform, bringing actionable data about course and learner performance to instructors and site operators. It is primarily a Tutor plugin that ties together data from the Open edX learning management system and Studio using open source tools to aggregate and transform learning traces into data visualizations.

See the [Aspects configuration documentation](https://docs.openedx.org/projects/openedx-aspects/en/latest/technical_documentation/how-tos/production_configuration.html) to learn about setting up Aspects for your production environment.
----
Do I have that right, @bmtcril ?

Reply:
Folks I want to keep this *laser focused* on dev/ops stuff if I can please.

Seems like there are no updates for me to make here? Yell if you disagree and I'll resolve tomorrow otherwise :)

Thanks everyone for chiming in! Appreciate all the feedback.

Reply:
I wouldn't generically point at Aspects configuration docs. However, I would mention anything operators who are _already_ running Aspects in Redwood need to do to upgrade to Sumac.

Reply:
And further, if it Just Works (TM), I would say that as well (eg, "Migration of Aspects from Redwood to Sumac requires no operator intervention")

Reply:
Thanks, @sarina  

@bmtcril, I know you've likely told me this before, but is there anything operators who are already running Aspects in Redwood need to do to upgrade to Sumac? Or should it just work with no operator intervention?

Reply:
There is no requirement to upgrade Aspects for Sumac, it versions separately from named releases. We can put in a suggestion that people upgrade to `v1.3.1` to get the latest functionality, however, and link the [upgrade doc](https://docs.openedx.org/projects/openedx-aspects/en/latest/technical_documentation/how-tos/upgrade.html).